tests/integration/test_akafka.py

```diff
@@ -26,11 +26,8 @@
     KafkaEventPublisher,
     KafkaEventSubscriber,
 )
-from hexkit.providers.akafka.testutils import (  # noqa: F401
-    ExpectedEvent,
-    KafkaFixture,
-    kafka_fixture,
-)
+from hexkit.providers.akafka.testutils import ExpectedEvent, KafkaFixture  # noqa: F401
```

Comment:
Or maybe we could import everything from `fixtures`? The fixtures module should just import everything useful liek ExpectedEvent, and declare it in its `__all__` list. Then we don't need two import statements.

Just and idea...

Reply:
Changed in [62c8334](https://github.com/ghga-de/hexkit/pull/59/commits/62c83347c2f2270183439246b4a8bdebafa7039e). Take a look and let me know if there is anything imported/not imported that should be changed. I just imported everything, even the constants from the S3 stuff.